src/utils/definition.js

```diff
@@ -30,4 +30,14 @@ export default class Definition {
     const effective = Math.ceil((get(definition, 'described.score', 0) + get(definition, 'licensed.score', 0)) / 2)
     return { tool, effective }
   }
+
+  /**
+   * Determine if a component doesn't have any data. In order to show in the UI in the list of Components
+   *
+   * @param {*} definition
+   * @returns {boolean}
+   */
+  static isDefinitionEmpty(definition) {
```

Comment:
This naming and commenting is too broad. There could be tons of other data, and just the source location is missing (actually that is the 90% case).